catalogue/webapp/pages/progress.tsx

```diff
@@ -43,10 +25,7 @@ const ProgressPage = () => (
         <div className="grid">
           <div className={`${grid({ s: 12, m: 11, l: 8, xl: 7 })}`}>
             <div className="body-text">
-              <ReactMarkdown
-                renderers={{ heading: HeadingRenderer }}
-                source={PROGRESS_NOTES}
-              />
+              <PROGRESS_NOTES />
```

Comment:
How does this get rendered properly now, without ReactMarkdown?

Reply:
Nope, using [Next MDX](https://mdxjs.com/getting-started/next).

[See here](https://github.com/wellcomecollection/wellcomecollection.org/pull/5734/files#diff-509bf115f04eb79bd3d10764253225520923d1ba25efbc3695834990b1e60a41R4-R6)

Reply:
Ah, just seen the changes to the next config